plugins/UsersManager/API.php

```diff
@@ -513,6 +514,22 @@ public function getUsersHavingSuperUserAccess()
     }
 
     /**
+     * Regenerate the token_auth associated with a user.
+     *
+     * @param   string  $userLogin
+     * @throws  Exception
+     */
+    public function regenerateTokenAuth($userLogin)
+    {
+        Piwik::checkUserHasSuperUserAccessOrIsTheUser($userLogin);
+
+        $this->model->updateUserTokenAuth(
+            $userLogin,
+            $this->createTokenAuth($userLogin)
+        );
+    }
+
+    /**
      * Updates a user in the database.
      * Only login and password are required (case when we update the password).
      * When the password changes, the key token for this user will change, which could break
```

Comment:
We can remove this comment now, and maybe mention that the token will only change if a legacy token is being used